action.yml

```diff
@@ -38,40 +38,48 @@ inputs:
 
   iac_type:
     description: |-
-      IaC template type. Currently only `terraform` is supported.
```

Comment:
You can remove "the default is" sentences from all of these. The generated documentation will automatically insert that.

Reply:
Done